src/test/java/teammates/test/cases/browsertests/InstructorSubmissionAdjustmentUiTest.java

```diff
@@ -78,9 +80,19 @@ public void testAdjustmentOfSubsmission() {
                      + student.toEnrollmentString();
         enrollPage.enroll(enrollString);
 
-        int numberOfNewResponses =
-                getAllResponsesForStudentForSession(student, session.getFeedbackSessionName()).size();
-        assertEquals(0, numberOfNewResponses);
+        // The operation should be retried as it might take a while for the submission adjustment to persist (especially
+        // on the live server), during which the pre-existing submissions and responses would result in a non-zero count.
+        RetryManager.runUntilSuccessful(new RetryableTaskReturns<Integer>("Assert zero new responses") {
```

Comment:
"Assert zero new responses" seems misleading, since it seems to imply new responses may appear and we are asserting that there are no new responses. "Assert non-relevant(/outdated) team responses are removed"?